tests/phpunit/CRM/Contact/BAO/QueryTest.php

```diff
@@ -658,6 +658,49 @@ public function testNonReciprocalRelationshipTargetGroupIsCorrectResults() {
     $this->assertEquals($contactID_b, $dao->contact_id, "Search query returns parent of contact A?");
   }
 
+  /**
+   * Relationship search with custom fields.
+   */
+  public function testReciprocalRelationshipWithCustomFields() {
+    $params = [
+      'extends' => 'Relationship',
+    ];
+    $customGroup = $this->customGroupCreate($params);
+    $customField = $this->customFieldCreate(['custom_group_id' => $customGroup['id']])['id'];
+    $contactID_a = $this->individualCreate();
+    $contactID_b = $this->individualCreate();
+    $this->callAPISuccess('Relationship', 'create', [
+      'contact_id_a' => $contactID_a,
+      'contact_id_b' => $contactID_b,
+      'relationship_type_id' => 2,
+      'is_active' => 1,
+      "custom_{$customField}" => 'testvalue',
+    ]);
+    $params = [
+      [
+        0 => 'relation_type_id',
+        1 => 'IN',
+        2 =>
+          [
+            0 => '2_a_b',
+          ],
+        3 => 0,
+        4 => 0,
+      ],
+      [
+        0 => "custom_{$customField}",
+        1 => '=',
+        2 => 'testvalue',
+        3 => 0,
+        4 => 0,
+      ],
+    ];
+
+    $query = new CRM_Contact_BAO_Query($params);
+    $dao = $query->searchQuery();
+    $this->assertEquals('2', $dao->N);
+  }
```

Comment:
```suggestion
    $this->assertEquals('2', $dao->N);
    $this->callAPISuccess('Relationship', 'delete', ['id' => $relationship['id']]);
    $this->callAPISuccess('Contact', 'delete', ['id' => $contactID_a, 'skip_undelete' => 1]);
    $this->callAPISuccess('Contact', 'delete', ['id' => $contactID_b, 'skip_undelete' => 1]);
    $this->callAPISuccess('CustomField', 'delete', ['id' => $customField, 'skip_undelete' => 1]);
    $this->callAPISuccess('CustomGroup', 'delete', ['id' => $customGroup]);
  }
```
@jitendrapurohit this seems to fix the reported test failure

Reply:
`$relationship` wasn't declared above, so I've pushed another commit fixing the test failures. This should pass the build now. Thanks.